### ISSUE TITLE
Context data not inherited by resource

### ISSUE THREAD

**Issue by danReynolds** [DataDog/dd-sdk-reactnative#513]
Per the docs here: https://docs.datadoghq.com/real_user_monitoring/browser/monitoring_resource_performance/
> RUM Resources inherit from all the context related to the active RUM View at the time of collection.

However maybe that's only applicable on web because we've found that our resource events inherit the View ID and name, but not custom context we added to the view and we aren't able to filter by custom view context data.

Is this a known limitation with the react-native-sdk specifically (on 1.0.0-rc5) or perhaps it's supported in a newer version? Tried upgrading but blocked by another issue: https://github.com/DataDog/dd-sdk-reactnative/issues/512

Thanks for your time!

**Comment by louiszawadzki** [comment]
Hi @danReynolds, thanks for reaching out!

So far we cannot reproduce the issue on the latest version. Could you specify which part of the context data you're missing specifically to help us troubleshoot?


Thanks a lot!

**Comment by danReynolds** [comment]
Thanks for the quick reply @louiszawadzki ! I'm passing extra context to the view like this:
```
DdRum.startView(
     'view_id',
     'view_name',
    { customField },
    Date.now()
);
```

but then when I look at the resource reports for this view, they don't have the customField context value on them. This is on 1.0.0-rc5 so maybe it works on latest, need to get through https://github.com/DataDog/dd-sdk-reactnative/issues/512 to try that

**Comment by louiszawadzki** [comment]
Hi @danReynolds, thanks a lot for clarifying!

Indeed, passing some context to a RUM View when creating it will not forward this context to children event like RUM Resources. It will only be attached to the RUM View.
If you want to have data attached to all your RUM events, use `DdSdkReactNative.setAttributes`.

The documentation you referred to is the documentation for our Browser SDK, which does not have a way to specify attributes for a RUM View, only global attributes (similarly to `DdSdkReactNative.setAttributes`), which I understand can be confusing.

If you want to find a way to filter your RUM Resources depending on the views, I encourage you to reach out to our [support team](https://www.datadoghq.com/support/) which will be able to assist you on how to use the Datadog platform for your specific need.

I'll close this issue and let the other one opened until we solve it. Feel free to reopen it if I missed something :)
